packages/fax/src/models/v3/enums.ts

```diff
@@ -21,11 +21,14 @@ export type ErrorType =
   | 'FAX_ERROR'
   | 'FATAL_ERROR'
   | 'GENERAL_ERROR'
-  | 'LINE_ERROR';
+  | 'LINE_ERROR'
+  | string;
 
-export type FaxBase64FileType = 'DOC' | 'DOCX' | 'PDF' | 'TIF' | 'JPG' | 'ODT' | 'TXT' | 'HTML' | 'PNG';
+export type FaxBase64FileType = 'DOCX' | 'PDF' | 'TIF' | 'JPG' | 'TXT' | 'HTML' | 'PNG' | string;
```

Comment:
Is there a ticket created toward Doc to highlight discrepancies between newdocs and oas-documentation ?
This modification is not reflecting spec file from oas-documentation

Reply:
No, I'll bring it to the agenda of the next meeting with the doc team